### PR TITLE
Document the filter on term option in deals.list

### PR DESCRIPTION
**Pull request by Kemosabert**
### ADDED
Option to filter on term in the `deals.list` endpoint

Note: I found the keys on which it actually searches in the `DbalDealRepository.php` file.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1349,6 +1349,7 @@ Get a list of deals.
     + Attributes (object)
         + filter (object, optional)
             + ids: `3f507137-e599-41d4-8625-0e2adba0be4c`,`ae97f6d3-0bb4-43d3-b13d-f06069d07e98` (array[string], optional)
+            + term: `Interesting deal` (string, optional) - Filters on the title, reference and customer's name
```
Comment (WouterSioen) [inline apiary.apib:1352]: I think `quotation number` is a more correct domain term than `reference`, right?

Summoning the deals team: @dietrichm @steftrenson

Comment (Kemosabert) [reply apiary.apib:1352]: Ah, I went with `reference` as I assumed that's the key it's refering to

Comment (WouterSioen) [reply apiary.apib:1352]: Going to double check if that's the same.

Comment (WouterSioen) [reply apiary.apib:1352]: Yep, it's the same!

**Review by WouterSioen — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Comment by dietrichm** [comment]
✅ Checked

**Review by JordyDeGroeve — APPROVED** [review]

**Pull request merged**